website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
+title: Introduction to FLINT
+author: Ruchi Pakhle
+author_url: https://github.com/Ruchip16
+author_title: Passionate programmer, coder & a web-developer 
+tags: [Intro to Flint, docusaurus]
```

Comment:
Keep tags as `[flint, moja-global]`. 

@aornugent can you suggest some more for the target audience?